web/src/styles/onboarding-wizard.css

```
@@ -0,0 +1,986 @@
+
+.onboarding-handoff-bubble {
+  padding: var(--space-3);
+  border: 1px solid var(--border);
+  border-radius: var(--radius-md);
+  background: var(--bg-warm);
+  color: var(--text);
+  font-size: var(--text-sm);
+  line-height: 1.5;
+  /* The mirrored issue can be long; keep it tidy inside the card. */
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)